### ISSUE TITLE
Cleanup for CreatePostingWithShifts

### ISSUE THREAD

**Issue by LenaNguyen** [uwblueprint/sistering#276]
Looks good to merge to unblock testing but there's a couple of TODOs that should be documented and/or addressed in a separate PR :)

1. Naming of `ShiftDataWithoutPostingId` type
2. `updatePosting` should now likewise include a shifts update
3. Unused mutations and methods (e.g. old bulk `createShifts`) and related types should be deleted
4. Shift creation conflict check

_Originally posted by @sherryhli in https://github.com/uwblueprint/sistering/pull/273#pullrequestreview-947844996_

Tech notes:
- rename type `ShiftDataWithoutPostingId` to `ShiftBulkRequestWithoutPostingId`

**Comment by MatoPlus** [comment]
> Unused mutations and methods (e.g. old bulk createShifts) and related types should be deleted

For this ticket. These related functions will be deprecated instead of deleted as they could still be useful for debugging purposes when testing locally

**Comment by MatoPlus** [comment]
> Unused mutations and methods (e.g. old bulk createShifts) and related types should be deleted

This will not be done since the updated resolvers are not 1:1 with old resolvers, can still be used independently for testing

> Shift creation conflict check
This is already done in the `builkGenerateTimeblocks` resolver
